site/student-lectures/description.rst

```diff
@@ -0,0 +1,92 @@
+********************************
+Student Lecture Task Description
+********************************
+
+* **Maximum Points**: 10
+* **Topic Selection Due Date**: October 11, 2023
+* **PR Merge Date**: At Least One Day Before the Lecture Date
```

Comment:
> Interesting strategy. I'm a little wary of forcing them to learn GitHub and git, but it's also fourth year students. Curious to see how it goes.

I'm hoping it goes fine. I already showed them how to do it in a lecture, so hopefully that's enough of a jumping off point to figure things out. 

> Probably worth doing a small readme which contain links to rst/sphinx docs, as well as GitHub stuff. But mainly for repo quirks, like where to add your lecture (makes sense to give them all a directory to house images and such), and how to add it to the overall index.

Good call. I did say "add under such and such a directory", but I should get more specific. 

> Worth noting, you're paying for the file hosting. May want to prevent them from pushing a data file or similar.

Another good call.